.github/workflows/ci.yml

```diff
@@ -123,7 +123,7 @@ jobs:
       - name: Build with Maven
         run: |
           set -o pipefail; mvn clean install -DskipTests | tee ./mvn.log; set +o pipefail
-          if [[ $(grep -c "overlapping classes" ./mvn.log) -gt 0 ]];then
```

Comment:
I think overlapping classes are shown per jar. So this change might hide some other overlapping classes in different jars